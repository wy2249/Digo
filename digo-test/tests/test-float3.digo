func testfloat(a float, b float) void {
  println("%f", a + b)
  println("%f", a - b)
  println("%f", a * b)
  println("%f", a / b)
  println("%f", a == b)
  println("%f", a == a)
  println("%f", a != b)
  println("%f", a != a)
  println("%f", a > b)
  println("%f", a >= b)
  println("%f", a < b)
  println("%f", a <= b)
}

func digo_main() void {
  c := 42.0
  d := 3.14159

  testfloat(c, d)
  testfloat(d, d)
}
