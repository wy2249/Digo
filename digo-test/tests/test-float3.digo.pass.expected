45.141590
38.858410
131.946780
13.369027
0.000000
1.000000
1.000000
0.000000
1.000000
1.000000
0.000000
0.000000
6.283180
0.000000
9.869588
1.000000
1.000000
1.000000
0.000000
0.000000
0.000000
1.000000
0.000000
1.000000
